### ISSUE TITLE
UX: Windows 10 Lock Screen Mouse Interruption

### ISSUE THREAD

**Issue by wykydtronik** [symless/synergy#5255]
I have 2 Windows 10 computers connected with Synergy. When both computers are locked, clicking the second lock screen to access the login causes the mouse to bounce back to the main Synergy server primary monitor. In order to enter or click on the Password field, I have to move my mouse back over to the second Win10 monitor. Once I enter the password and hit enter, the mouse bounces back to the main Win10 monitor. Although a minor bug, this is an inconvenient user experience that alternative programs don't have.

**Comment by the-wes** [comment]
this is due to Synergy automatically restarting in order to gain the elevated privileges required to interact with the lock screen / login screen. Restarting involves disconnecting, which involves the mouse moving to the server. what Synergy should do is, specifically when it's restarting due to permissions problems, notify the server of this so the server mouse remains hidden, and focus is moved back to the client automatically once it reconnects.

**Comment by Aethalides** [comment]
Not at all, I'd say, get the new connection up and running in the background and only disconnect the previous one when the new one is ready. As it is, One needs to be very careful typing in ones password, because click in the password box, and the disconnect happens, so the focus goes back to your previous 'screen' which can be a chat window. Without noticing this, you proceed to type in your password...

Is it, for instance, not possible to do this reloading in advance, i.e. lockscreen is active => immediate reload for priv elevation.

Or, to do it, in the background, only switching to the new one when it is ready? [that may be more difficult come to think of it]

**Comment by nlyan** [comment]
If you change Elevate Mode in settings to 'Always' in 1.8.3 and onward, this disconnect should not occur.
